website/content/en/docs/contribution-guidelines/testing/local-changes.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Testing Changes Locally
+linkTitle: Local Changes
+weight: 40
+---
+
+## Testing Changes Locally
+
+If your changes are in the SDK commands then you just need to run `make install` to be able to use an SDK binary built from the source code and then test locally it. Also, see that you can run `operator-sdk version` to check what is the commit used to built it. 
+
+However, If the change performed is NOT in the scaffold files or sdk commands then, is required to build an new image with the changes done to test it locally in a POC operator project. In this way, by using this dev image in an operator project locally we will be able to check the changes made for the Ansible/Helm based-operator. And then, for the GO based-operators, will be required ensure that you are import your version of code implementation to be used in the POC operator project were it will be checked.  
+
```

Comment:
Supplement the info by adding GO as well.

Reply:
I don't think this guide belongs in "testing". I would assume that "testing" means running tests and would be surprised that I needed to read in this area just to get my local code changes into my environment.

Reply:
Oks. moved to the main menu with local docs